testsuite/features/support/api_test.rb

```diff
@@ -65,16 +77,24 @@ def call(name, *params)
 
 # Derived class for an XML-RPC test
 class ApiTestXmlrpc < ApiTest
+  ##
+  # It creates a new instance of the XmlrpcClient class, and assigns it to the @connection instance variable
+  #
+  # Args:
+  #   host: The hostname of the server.
   def initialize(host)
     super
     @connection = XmlrpcClient.new(host)
   end
 
+  ##
   # during XML-RPC tests, dates are XMLRPC::DateTime's
```

Comment:
I'm not sure how I feel about 2 methods with the exact same name in the same place, but aditionally to that, just as its twin, we should also document this one in a more descriptive way @Bischoff

Reply:
It overrides the method, as it might made differently for HTTP and XMLRPC.
I'm fine with the current description.